### PR TITLE
Add xyz.tytanium.DoorKnocker

### PR DESCRIPTION
**Pull request by tytan652**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [ ] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
 - `"--talk-name=org.freedesktop.Flatpak"` is added to allow DoorKnocker to get xdg-desktop-portal version directly from the executable
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. 
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by tytan652** [comment]
bot, build xyz.tytanium.DoorKnocker

File: flathub.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
[the remark this answers is not in the document]
Comment (tytan652) [reply flathub.json:1]: I know, but I really wanted to make it explicit.
If I must remove it, tell me.

[the remark this answers is not in the document]
Comment (hfiguiere) [reply flathub.json:1]: yes, please remove.

[the remark this answers is not in the document]
Comment (tytan652) [reply flathub.json]: Done

**Pull request closed without merging**